app/src/main/java/ca/marshallasch/veil/CommentListAdapter.java

```diff
@@ -84,12 +85,18 @@ public void update(List<DhtProto.Comment> comments) {
     public void onBindViewHolder(@NonNull ViewHolder holder, final int position) {
 
         DhtProto.Comment comment = commentList.get(position);
-        String aythorName = comment.getAnonymous() ? context.getString(R.string.anonymous) : comment.getAuthorName();
+        String authorName = comment.getAnonymous() ? context.getString(R.string.anonymous) : comment.getAuthorName();
 
-        holder.authorName.setText(aythorName);
+        holder.authorName.setText(authorName);
         holder.content.setText(comment.getMessage());
-        holder.date.setText(Util.timestampToDate(comment.getTimestamp()).toString());
 
```

Comment:
Comment what this is for "Retrieving relative timestamp." or something like that.

Reply:
fixed in commit 8a05ddb